.github/workflows/build_docs.yml

```diff
@@ -0,0 +1,42 @@
+# Please keep in sync with /environment.yml
```

Comment:
What is it that has to be kept in sync? Is there a way to ensure that it is?

Reply:
Oh that's a copy-paste relic from the main build action (where it refers to setting up the environment where we run Fab). Will remove.